src/pages/publication/files_and_license_tab/FileCard.tsx

```diff
@@ -172,7 +167,11 @@ const FileCard: React.FC<FileCardProps> = ({ file, removeFile, toggleLicenseModa
                         helperText={<ErrorMessage name={field.name} />}
                         label={t('files_and_license.license')}
                         onChange={({ target: { value } }) =>
-                          setFieldValue(field.name, { identifier: value as LicenseNames, labels: { nb: value } })
+                          setFieldValue(field.name, {
+                            type: 'License',
+                            identifier: value as LicenseNames,
+                            labels: { nb: value },
```

Comment:
hvordan vet vi hvilket språk det skal være på? eller bruker vi kun norsk nå?

Reply:
vi aner ikke :D gikk bare minste motstands vei
burde sikkert basere oss på språk valgt for user eller primærsspråk for publikasjon en gang i fremtiden.